index.js

```diff
@@ -55,31 +61,45 @@ class CarmiRoot extends React.Component {
 }
 
 class CarmiObserver extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
This is needed to avoid the following warning:
`Warning: “CarmiObserver” uses “getDerivedStateFromProps” but its initial state is undefined. This is not recommended. Instead, define the initial state by assigning an object to “this.state” in the constructor of “CarmiObserver”. This ensures that “getDerivedStateFromProps” arguments have a consistent shape.`